main.nf

```diff
@@ -425,28 +564,32 @@ if( params.aligner == 'bismark' ){
               multicore = "--multicore $ccore"
             }
         }
-        if( params.singleEnd ) {
+        if( params.single_end ) {
             """
-            bismark \\
+            bismark $aligner \\
                 --bam $pbat $non_directional $unmapped $mismatches $multicore \\
                 --genome $index \\
-                $reads
+                $reads \\
```

Comment:
You could simplify that.
with something like 
```
input = params.single_end ? reads : "-1 ${reads[0]} -2 ${reads[1]}"
"""
             bismark $aligner \\
                 --bam $pbat $non_directional $unmapped $mismatches $multicore \\ 
                 --genome $index \\ 
                 $input \\
                 $soft_clipping \\
                 $splicesites
```

Reply:
Agreed, that's much nicer 👍

Reply:
Same with other bismark commands. I guess I wrote this quite a long time ago 😅